mix.exs

```diff
@@ -412,6 +412,7 @@ defmodule EMQXUmbrella.MixProject do
           emqx_bridge_timescale: :permanent,
           emqx_bridge_sqlserver: :permanent,
           emqx_oracle: :permanent,
+          emqx_redis: :permanent,
```

Comment:
I think we need to add the `emqx_bridge_redis` as well (and in rebar.config) to include it in the release.

Reply:
It is already added as there was and app template for redis before this PR. See higher up in the list.